indexing/run_redis.sh

```diff
@@ -5,6 +5,9 @@ set -x
 
 if [ -f  ../../valkey/src/valkey-server ]; then
     ../../valkey/src/valkey-server ./indexing.conf
-else
+elif [ -f ../../redis/src/redis-server ]; then
     ../../redis/src/redis-server ./indexing.conf
+else
+    /usr/bin/redis-server ./indexing.conf
```

Comment:
I'm not totally against that, but I always use the latest version of redis (or valkey now), and that includes the latest commands. Using the version from the packages often gives you a very old version that may or may not work.

Can you print a message saying that it is unsupported and may cause runtime issues?

Reply:
I added the message for both cases. Also (tried) to fix the python test raising the variable type. Can you approve test workflows again?